cheevos/cheevos.c

```diff
@@ -1382,12 +1382,10 @@ static void rcheevos_client_login_callback(int result,
 
 static void rcheevos_finalize_game_load(rc_client_t* client)
 {
-#if defined(HAVE_GFX_WIDGETS) /* We always want badges if widgets are enabled */
-   bool want_badges     = true;
-#else
```

Comment:
Is there some other compile flag (maybe `HAVE_THREADS`) that we can use to manage this behavior? The stuttering doesn't happen on Windows or Linux, so there's no reason to disable the badges there.

Reply:
Are badges disabled with this change? That wasn't my intention and I'm not seeing it.

Reply:
Now explicitly, but you're enabling the option for all systems, even those not affected by the stuttering.

Overall, it's not a bad change, but the biggest complaint I hear about RetroArch is that there's too many menu options.

Reply:
I agree both with the complaint and with you, but I see this as an exception - this is buried within the context of retro achievements settings.

What's actually bad is that I fear that people have it disabled in their configs, but it's currently being overridden by this hardcoded assignment to true.